app/bundles/EmailBundle/EventListener/PointSubscriber.php

```diff
@@ -131,4 +148,55 @@ public function onEmailSend(EmailSendEvent $event)
 
         $this->pointModel->triggerAction('email.send', $event->getEmail(), null, $lead);
     }
+
+    /**
+     * @param PointChangeActionExecutedEvent $changeActionExecutedEvent
+     */
+    public function onEmailOpenPointChange(PointChangeActionExecutedEvent $changeActionExecutedEvent)
+    {
+        $action = $changeActionExecutedEvent->getPointAction();
+
+        if ($action->getType() != 'email.open') {
```

Comment:
Here we can use the `checkContext` that I suggested be added to the PointChangeActionExecutedEvent.